tutorials/tutorial01_sumo.ipynb

```diff
@@ -253,7 +255,7 @@
    "source": [
     "from flow.core.params import SumoParams\n",
     "\n",
-    "sumo_params = SumoParams(sim_step=0.1, render=True)"
+    "sumo_params = SumoParams(sim_step=0.1, render=True, emission_path='data')"
```

Comment:
can we add a snapshot of the emission file at the end plus a small description? I think this would be very helpful for people that want to understand how to visualize